charts/kyverno/templates/cleanup-controller/clusterrole.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.cleanupController.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "kyverno.cleanup-controller.deploymentName" . }}
+  labels:
+    {{- include "kyverno.cleanup-controller.labels" . | nindent 4 }}
```

Comment:
` app: kyverno` is added to all other Kyverno resources, do we need it?

Reply:
Nope, it will confuse kyverno service I think.